### PR TITLE
Add ruby_namespace option to PubSub

### PR DESCRIPTION
**Pull request by blowmage**

### CODE REVIEW

**Comment by jbolinger** [comment]
These changes look correct, but they need to be made internally or the next sync may revert it. @dazuma should this be updated? (same for #531)

File: google/pubsub/v1/pubsub.proto [hunk 1]
```diff
@@ -30,6 +30,7 @@ option java_multiple_files = true;
 option java_outer_classname = "PubsubProto";
 option java_package = "com.google.pubsub.v1";
 option php_namespace = "Google\\Cloud\\PubSub\\V1";
+option ruby_namespace = "Google::Cloud::PubSub::V1";
```
Comment (andreamlin) [inline google/pubsub/v1/pubsub.proto]: [drive-by question]
What is the rationale for including the `ruby_namespace` now, if the v1 proto library has already been generated and in existence for a while?

Comment (blowmage) [reply google/pubsub/v1/pubsub.proto]: Mostly consistency with other languages that already use "PubSub" instead of "Pubsub". We are trying to get the `google-cloud-pubsub` gem to GA, so if we are going to make this change it should happen before then.

Comment (andreamlin) [reply google/pubsub/v1/pubsub.proto]: I see.

**Comment by blowmage** [comment]
Can we get an update on this?

**Comment by dazuma** [comment]
@jbolinger Yeah we should move forward on this. I can handle it, as we need to push the proto change through, and introduce backward-compatibility aliasing in the synth script.

**Comment by sduskis** [comment]
@blowmage, can this be closed?

**Comment by blowmage** [comment]
Yes, I believe this PR can be closed.

#531 is not yet changed and should remain open.

**Comment by blowmage** [comment]
Spoke too soon. this change has been made to V1, but not V1beta2.

https://github.com/googleapis/googleapis/blob/master/google/pubsub/v1beta2/pubsub.proto#L21-L24

**Comment by sduskis** [comment]
@blowmage, these changes have to happen in a different place.  I started that process.

**Comment by blowmage** [comment]
The google-cloud-pubsub gem doesn't have v1beta2, so re-closing.
